File: second_semister/oop/week_07/MUL/mid.py
n = input()

for _ in range(n):
    a = input()
    b = input()
    c = input()

    l = [a, b, c]

    big = a
    small = c

    for a in l:
        if a > big:
            big = a

    for a in l:
        if a < big:
            small = a

    for a in l:
        if not (a == big or small == a):
            print(a)
            break

File: second_semister/oop/week_07/MUL/mid.cpp
#include <bits/stdc++.h>
using namespace std;
int main()
{
    int n;
    cin >> n;

    while (n--)
    {
        int a, b, c;
        int big = a;
        int small = a;
        cin >> a >> b >> c;

        // small
        if (a < b && a < c)
        {
            small = a;
        }
        if (b < a && b < c)
        {
            small = b;
        }
        if (c < b && c < a)
        {
            small = c;
        }
        // big
        if (a > b && a > c)
        {
            big = a;
        }
        if (b > a && b > c)
        {
            big = b;
        }
        if (c > b && c > a)
        {
            big = c;
        }

        if (a > small && a < big)
            cout << a << endl;
        else if (b > small && b < big)
            cout << b << endl;
        else if (c > small && c < big)
            cout << c << endl;
    }

    return 0;
}